webroot/js/app.js

```diff
@@ -398,17 +397,33 @@ export default class App extends Component {
               ${tag}
             </li>
           `
-        );
-
-    const socialIconsList = !socialHandles.length
-      ? null
-      : socialHandles.map(
-          (item, index) => html`
-            <li key="social${index}">
-              <${SocialIcon} platform=${item.platform} url=${item.url} />
-            </li>
-          `
-        );
+        )
+      : null;
+
+    const socialIconsList = (function () {
```

Comment:
Since it's getting clunky, why don't we extract this into its own component like a <SocialList> (or a socialist? jk), that we can pass the `socialHandles` into, and let the component do the rest of the work.

Reply:
I did not feel like inventing new components for this, but now that you say it, yess.

Reply:
I have moved the icons list to its own component now.

Reply:
Sorry, just saw this update! 
Can I be picky and request that the file name be in kebab-case, instead of camelCase, to be consistent with the other files in /webroot? 
Otherwise looks good! :shipit: